pkg/cmd/pr/create/create.go

```diff
@@ -315,7 +337,7 @@ func createRun(opts *CreateOptions) (err error) {
 			ghrepo.FullName(ctx.BaseRepo))
 	}
 
-	if opts.FillVerbose || opts.Autofill || opts.FillFirst || (opts.TitleProvided && opts.BodyProvided) {
+	if !opts.EditorMode && (opts.FillVerbose || opts.Autofill || opts.FillFirst || (opts.TitleProvided && opts.BodyProvided)) {
```

Comment:
I wonder how this logic could be broken down to be easier to reason. 🤔 

For comparison, here's how the `gh issue create` logic corollary:

https://github.com/cli/cli/blob/e2275d7ef62c13fe63c80dd204e2a64417be2d8c/pkg/cmd/issue/create/create.go#L117-L124

Reply:
My first thinking is to use `opts.interactive`
```
opts.Interactive = !(opts.FillVerbose || opts.Autofill || opts.FillFirst) && !(opts.TitleProvided && opts.BodyProvided)
```

---

L211 can be changed
https://github.com/cli/cli/blob/e2275d7ef62c13fe63c80dd204e2a64417be2d8c/pkg/cmd/pr/create/create.go#L211

```go
if !opts.IO.CanPrompt() && !opts.WebMode && opts.Interactive 
```

---

And also L340:
https://github.com/cli/cli/blob/e2275d7ef62c13fe63c80dd204e2a64417be2d8c/pkg/cmd/pr/create/create.go#L340
```go
if !opts.EditorMode && !opts.Interactive {
```
---
However, I'm concerned that `opts.Interactive` in `pr/create` might differ from `opts.Interactive` in `issue/create` because the condition for `opts.Interactive` in `pr/create` does not include `opts.Editor`, while it does in `issue/create`.

Reply:
> However, I'm concerned that opts.Interactive in pr/create might differ from opts.Interactive in issue/create because the condition for opts.Interactive in pr/create does not include opts.Editor, while it does in issue/create.

That makes sense; PR creation has some unique challenges that issue doesn't have.

While bringing more of these 2 commands into alignment is a great long term goal, that is definitely outside of the scope of this PR.  Just musing about here + now as well as later.